draft-iab-m-ten-workshop.md

```diff
@@ -317,35 +317,35 @@ After an introduction that covered the goals of the workshop and the starting qu
+A queriable relay mechanism could preserve network management functions that are disrupted by encryption, such as TCP optimisation, quality of service, zero-rating, parental controls, access control, redirection, content enhancement, analytics and fraud prevention. Instead of encrypted communication between only two ends and passive observation by all on-path elements, intermediate relays could be trusted parties with limited information for the purposes of collaboration between in-network intermediary services' support.
 
 ### Visible, optional network management
 
-In encrypted communications, out of all of the possible network management functions that might be ameliorated by proxying the ability to control congestion has been researched in depth. These techniques are realized based on TCP performance enhancing proxies (PEP) that either entirely intercept a TCP connection or interfere with the transport information in the TCP header. However, beside the challenge that new encrypted protocol limited any such in-network interference, these techniques can also have negative impact on the evolvability of these protocols. Therefore, instead on manipulating existing information, a new approaches was presented  where additional information is send using a so-called side-car protocol independent of the main transport protocol that is used end-to-end {{WELZL}}. E.g. side car information can contain additional acknowledgements to enable in-network local retransmission faster end-to-end retransmission by reducing the signaling round trip time.
+In encrypted communications, out of all of the possible network management functions that might be ameliorated by proxying, the ability to control congestion has been researched in depth. These techniques are realized based on TCP performance enhancing proxies (PEP) that either entirely intercept a TCP connection or interfere with the transport information in the TCP header. However, the challenge that the new encrypted protocol will limit any such in-network interference, these techniques can also have a negative impact on the evolvability of these protocols. Therefore, instead of manipulating existing information, a new approache was presented  where additional information is send using a so-called side-car protocol independent of the main transport protocol that is used end-to-end {{WELZL}}. E.g. side car information can contain additional acknowledgements to enable in-network local retransmission faster end-to-end retransmission by reducing the signaling round trip time.
```

Comment:
```suggestion
In encrypted communications, out of all of the possible network management functions that might be ameliorated by proxying, the ability to control congestion has been researched in depth. These techniques are realized based on TCP performance enhancing proxies (PEP) that either entirely intercept a TCP connection or interfere with the transport information in the TCP header. However, despite the challenge that the new encrypted protocol will limit any such in-network interference, these techniques can also have a negative impact on the evolvability of these protocols. Therefore, instead of manipulating existing information, a new approache was presented  where additional information is send using a so-called side-car protocol independent of the main transport protocol that is used end-to-end {{WELZL}}. E.g. side car information can contain additional acknowledgements to enable in-network local retransmission faster end-to-end retransmission by reducing the signaling round trip time.
```